app/screens/bottom_sheet/index.tsx

```diff
@@ -225,9 +225,9 @@ const BottomSheet = ({
             onClose={close}
             bottomInset={insets.bottom}
         >
-            <BottomSheetView style={styles.view}>
+            <View style={styles.view}>
```

Comment:
This does not look right to me, specially after reading https://gorhom.dev/react-native-bottom-sheet/usage

Reply:
@enahum I'm trying everything.

This is not reproducible on debug mode, or on simulator.  Creating a release version locally doesn't reproduce the issue.

So I have to create a draft like this with experiments and build with ci to get the ipa built.

Reply:
@enahum at least it's no longer dismissing automatically, but the bottom sheet is not popping to the correct spot.  I can probably fix that, but I'd like to investigate more tomorrow.

https://github.com/user-attachments/assets/474c5aad-8e01-4127-a562-9b92bbfc97b9

Reply:
cc @lindalumitchell

Reply:
@enahum FYI: I reverted this change